libkbfs/folder_branch_ops.go

```diff
@@ -3142,13 +3149,19 @@ func (fbo *folderBranchOps) reembedForFBM(ctx context.Context,
 
 type applyMDUpdatesFunc func(context.Context, *lockState, []*RootMetadata) error
 
+var errUnmerged = errors.New("fbo is on an unmerged local revision")
```

Comment:
Sounds good. There's an [`errShutdownHappened`](https://github.com/keybase/kbfs/blob/d8cd1200bde1ee33a2b9909bdf6925501dfac560/libkbfs/folder_branch_ops.go#L3730) as well. Should I change that as well?